tests/test_txp_mau_dau.py

```diff
@@ -0,0 +1,21 @@
+from mozetl.testpilot.txp_mau_dau import get_experiments
+from utils import get_resource
+import requests
+import json
+
+
+class MockRequest():
+    def raise_for_status(self):
+        pass
+
+    def json(self):
+        return json.loads(get_resource('txp_experiments.json'))
+
+
+def test_get_experiments(monkeypatch):
+    def mock_get(url):
+        return MockRequest()
+    monkeypatch.setattr(requests, 'get', mock_get)
+    # There are 14 total experiments in the file but one should be filtered out
+    # due to not having an addon_id
```

Comment:
Why do we want to ignore experiments without addons? Is it because we do not care about these experiments or because we can't identify active users without using an addon_id? Can you add a comment explaining why we don't care or why we're making this compromise?